README.md

```diff
@@ -28,10 +28,22 @@ sbt new apache/daffodil-schema.g8
 ```
 
 This command prompts for various properties and creates a bare-bones directory
-structure that follows the recommended [DFDL Schema Project Layout]. This
-includes git and sbt configuration files, a basic DFDL schema file, and [TDML]
+structure that follows the recommended [DFDL Schema Project Layout] and most recent
+best practices from the [DFDL Schema Style Guide]. This
+includes git and sbt configuration files, basic DFDL schema files, and [TDML]
 and test files.
 
+Once you have generated your schema directory, typing `sbt test` will work and 
+run a TDML test. Of course, the initial DFDL schema has no fields, the test data file is empty, and 
+the expected XML Infoset file is an empty element. But, it's working and you can
+keep it working as you fill in the details of the schema and add tests. 
+
+The schema project is set up to use the `daffodil-sbt` plugin for SBT as part of the
+SBT build and test. 
+This provides capabilities to compile your DFDL schema into binary form and package it for 
+distribution. 
+You can read more about `daffodil-sbt` here: [Apache daffodil-sbt Plugin]
+
```

Comment:
I just realized we still list the "daffodil_version" property below, that is no longer need since this was updated to use the sbt-daffodil plugin for specifying daffodil version.. Could you update this readme while you're at it?

Reply:
Will do.